alot/crypto.py

```diff
@@ -223,16 +223,7 @@ def decrypt_verify(encrypted, session_keys=None):
             pass
 
     ctx = gpg.core.Context()
-    try:
-        plaintext, _, verify_result = ctx.decrypt(encrypted, verify=True)
-        sigs = verify_result.signatures
-    except gpg.errors.GPGMEError as e:
-        raise GPGProblem(str(e), code=e.getcode())
-    except gpg.errors.BadSignatures as e:
-        plaintext, _, _ = ctx.decrypt(encrypted, verify=False)
-        sigs = e.result.signatures
-
-    return sigs, plaintext
+    return _decryp_verify_with_context(ctx, encrypted)
```

Comment:
shouldn't that be `_decrypt_verify_with_context`?

Reply:
I agree, it was a typo.